app/models/exchange_rate_country_currency.rb

```diff
@@ -1,3 +1,24 @@
 class ExchangeRateCountryCurrency < Sequel::Model(:exchange_rate_countries_currencies)
   plugin :timestamps, update_on_create: true
+  plugin :auto_validations, not_null: :presence
+  plugin :dirty
```

Comment:
Havent seen this before just looking into 👀

Reply:
Useful for working out if a column has changed